artifacts/source_type.py

```diff
@@ -165,6 +165,13 @@ class WindowsRegistryKeySourceType(SourceType):
 
   TYPE_INDICATOR = definitions.TYPE_INDICATOR_WINDOWS_REGISTRY_KEY
 
+  VALID_PREFIXES = [
+      r'HKEY_LOCAL_MACHINE',
+      r'HKEY_USERS',
+      r'HKEY_CLASSES_ROOT',
+      r'%%current_control_set%%',
```

Comment:
For those following along, it is valid for now, but we should change it:
https://github.com/ForensicArtifacts/artifacts/issues/120